bitnami/mongodb/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.20.5
+version: 10.20.6
```

Comment:
As this is adding a feature, could you bump the minor version?

Reply:
Thanks a lot for your review. I just updated it !